src/pages/training.js

```diff
@@ -105,6 +77,7 @@ export const query = graphql`
         }
         courses {
```

Comment:
the graphql query for the `courses` field just needs to include the `name`, `id` and `slug` properties, right? The rest are already fetched by the course page query

Reply:
True.